uw-frame-components/portal/widgets/partials/type__action-items.html

```diff
@@ -0,0 +1,37 @@
+<div class="widget-body action-items">
+  <!-- No action items to display -->
+  <div layout="row" layout-align="center center" ng-show="actionItems.length == 0">
+    <!-- Loading case -->
+    <loading-gif data-object="actionItems" ng-show="loading"></loading-gif>
+
+    <!-- Error case -->
+    <div class="error" layout="row" layout-align="start center" ng-show="error">
+      <div><md-icon class="md-warn">warning</md-icon></div>
+      <span>There was an issue loading the information. Click See all for more info.</span>
+    </div>
+  </div>
+
+  <!-- Has action items to display -->
+  <div ng-show="actionItems.length > 0">
+    <md-list class="widget-list md-2-line">
+      <md-list-item ng-repeat="item in actionItems | limitTo:itemsLimit" ng-click="goToAction(item.actionUrl)">
+        <div class="action-item__quantity" md-colors="{background:'primary'}">
+          <span>{{ item.quantity }}</span>
+        </div>
+        <div class="md-list-item-text">
+          <p ng-if="item.quantity == 1">{{ item.textSingular }}</p>
+          <p ng-if="item.quantity != 1">{{ item.textPlural }}</p>
```

Comment:
Some additional context: The "textSingular" and "textPlural" widget config options bubbled up during the grooming discussion for this story, so I went with that. 

I think extracting the formatting out of the application logic is a great idea, and something we should do _eventually_, but I think it falls outside the scope of this particular story (because we should do it everywhere).